lib/jira/resource/worklog.rb

```diff
@@ -8,8 +8,16 @@ class Worklog < JIRA::Base
       has_one :author, :class => JIRA::Resource::User
       has_one :update_author, :class => JIRA::Resource::User,
                               :attribute_key => "updateAuthor"
-      belongs_to :issue
       nested_collections true
+
+      def self.all(client, options = {})
+        url = client.options[:rest_base_path] + "/issue/#{options[:issue].key}/worklog"
+        response = client.get(url)
+        json = parse_json(response.body)
+        json['worklogs'].map do |worklog|
+          client.Worklog.build(worklog)
```

Comment:
What about using new?
```ruby
        json['worklogs'].map do |attrs|
          self.new(client, {:attrs => attrs})
        end
```

This would allow you to put back `issue` and probably make the tests green again